### ISSUE TITLE
Switch from logo of source from url to base64 img

### ISSUE THREAD

**Issue by BirdInFire** [faldez/tanoshi-extensions#34]
I would like to ask to switch from logo of source from url to base64 img, like that into tanoshi if the website where the logo is stored is blocked (ex: nhentai) the logo is shown and is not broken.

On the privacy perspective it remove a connection from the client to the source so only the server is known from the source (and not the client who open the page of the source (witch trigger the download of the picture from their browser)).

Another way to fix this is to "proxy" this picture, but i find base64 picture a way more easy way to fix this.

**Comment by faldez** [comment]
yeah, it's useful to avoid cors too. what do you think is better? convert the the image to base64 or just commit the file directly to git and serve from git ?

**Comment by BirdInFire** [comment]
> yeah, it's useful to avoid cors too. what do you think is better? convert the the image to base64 or just commit the file directly to git and serve from git ?

depend of the objective, to be more light the git, to be more "less connection is better" base64, in both case i propose this to avoid a potential trigger of a security app in a more monitored network, (for instance a user flagged as porn reader when he open up the settings page during it's midday pause on his phone at work).

but for me (since the app must be connected to work) the better is git, it's just change the url.

**Comment by faldez** [comment]
you can change the url to base64 too I think. since the browser just set `cover_url` to `<image src="cover_url"/>`

**Comment by BirdInFire** [comment]
> you can change the url to base64 too I think. since the browser just set `cover_url` to `<image src="cover_url"/>`

true, but i can say for cor management it does not change much (https://url of git, or base64: (who will not change after)) so i will say take the one who is easiest to manage for you.
